packages/mui-joy/src/Avatar/Avatar.test.js

```diff
@@ -0,0 +1,266 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { createRenderer, describeConformance, fireEvent } from 'test/utils';
+import Avatar, { avatarClasses as classes } from '@mui/joy/Avatar';
+import { unstable_capitalize as capitalize } from '@mui/utils';
+import { spy } from 'sinon';
+import PersonIcon from '../internal/svg-icons/Person';
+
+describe('<Avatar />', () => {
+  const { render } = createRenderer();
+
+  describeConformance(<Avatar />, () => ({
+    classes,
+    inheritComponent: 'div',
+    render,
+    muiName: 'MuiAvatar',
+    refInstanceof: window.HTMLDivElement,
+    testComponentPropWith: 'span',
+    skip: [
+      'themeVariants',
+      'classesRoot',
+      'componentsProp',
+      'themeDefaultProps',
+      'themeStyleOverrides',
+    ],
+  }));
+
+  describe('prop: variant', () => {
+    it('contained by default', () => {
+      const { getByTestId } = render(<Avatar data-testid="root" />);
+
+      expect(getByTestId('root')).to.have.class(classes.variantContained);
```

Comment:
```suggestion
      expect(getByTestId('root')).to.have.class(classes.variantLight);
```
You might have to update this one because of #31826.